gravitee-am-model/src/main/java/io/gravitee/am/model/RateLimit.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.model;
+
+import java.util.Date;
+
+/**
+ * @author Ashraful Hasan (ashraful.hasan at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+
+public class RateLimit {
+    private String id;
+    private String userId;
```

Comment:
I think we should also add referenceId (initialized with the domainID) and referenceType (initialized to ReferenceType.DOMAIN) in order to be able to delete all the rate_limit by domain

Reply:
referenceId and referenceType field added